src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/default-extension.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['services/husky/util', 'services/husky/translator'], function(util, translator) {
```

Comment:
1. i would keep this "javascript-convention" constructor uppercase and instances lowercase ... makes it for me more readable.
2. in a extension we should not rely on other extensions. the more explizite dependency with the services is better in extensions